### ISSUE TITLE
fading not quite right

### ISSUE THREAD

**Issue by drjwbaker** [datapraxis/sourcecaster#3]
So I added a new button 'list files and give them IDs' and the fading is a bit (test it and you will see what I mean..). I managed to break the fading last time and you - @thomasgpadilla - fixed it. But I don't know how you fixed it. Could you point out how please so I don't just break it every time. Ta!

**Comment by thomasgpadilla** [comment]
Hey @drjwbaker - looks like its working! Were you able to sort it out?

**Comment by drjwbaker** [comment]
No. It just seems to be working now. Weird... Is the explanation okay? I was rushing it during half-time of watching a rugby match (the idea to reuse this script here came to me, unawares, midway through the first half).
